src/stp/new_plays/AttackingPass.cpp

```diff
@@ -77,8 +77,20 @@ void AttackingPass::calculateInfoForRoles() noexcept {
     // TODO: compute the passing position
     const Vector2 passingPosition = calculatePassLocation();
 
+    auto ball = world->getWorld()->getBall().value();
+    std::optional<Vector2> intersection;
+
+    if ((ball->getPos() - passingPosition).length() <= 2.0 && ball->getVelocity().length() > 0.1) {
+        intersection = Line(ball->getPos(), ball->getPos() + ball->getVelocity()).intersects(
+            Line(passingPosition, field.getTheirGoalCenter()));
+    }
+
```

Comment:
If I understood correctly, this code assumes the ball will travel over the line between the passposition and their goal. This is not necessarily true, we might want a failsafe for if the shot was much softer than what we expected

Reply:
true